CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# main
+## Bug fixes 🐞
+* Change CopyOnWriteArrayList to CopyOnWriteArraySet in the sdk to hold listeners. ([1183](https://github.com/mapbox/mapbox-maps-android/pull/1183))
```

Comment:
perhaps add more color that duplicates will be eliminated? however not sure here